### DISCUSSION TITLE
How many participants in this dataset?

### DISCUSSION THREAD

**Discussion by WKQ9411** [AshrithSagar/EEG-Imagined-speech-recognition#2] [General]
Hi, Your work is impressive. 
I noticed that the original paper of KaraOne dataset mentioned that there were12 participants and data from 4 of them were discarded, but it seems that the paper did not mention which 4 participants' data were removed. In addition, why the website has published data for 14 participants? 
I'm quite confused about that. Could you kindly explain why the published data seems to include more participants than originally mentioned and is the 4 discarded data still in this 14 sets of data?

**Comment by AshrithSagar** [comment]
Hi, Thanks for the comment.

Yeah, they don't mention those details as to which of the participants were discarded.
The published dataset includes data from 14 participants, the specific identities of the discarded participants are not provided in the original paper or on the website.

If it helps, regarding the details on the subjects, I found the following to be consistent with what they mention on the website.
| Subject | Epochs shape |
|:-------:|:------:|
|   MM05  | (165,) |
|   MM08  | (131,) |
|   MM09  | (132,) |
|   MM10  | (132,) |
|   MM11  | (132,) |
|   MM12  | (132,) |
|   MM14  | (132,) |
|   MM15  | (132,) |
|   MM16  | (132,) |
|   MM18  | (132,) |
|   MM19  | (132,) |
|   MM20  | (132,) |
|   MM21  | (132,) |
|   P02   | (165,) |
